submissions/factorio_draftsman/main.md

```diff
@@ -0,0 +1,211 @@
+* **Draftsman is easy to use.** Designed from the start to be simple and (most importantly) self-documenting, Draftsman allows you to manipulate blueprints and entities by auto-completable attributes and methods.
+* **Draftsman is well-documented.** Every function, method, attribute, and class is documented and crosslinked at it's [readthedocs site](https://factorio-draftsman.readthedocs.io/en/latest/?badge=latest). In addition, tutorials and supplementary material is provided, as well as a whole host of [example programs](https://github.com/redruin1/factorio-draftsman/tree/main/examples) to help illustrate how Draftsman works.
+* **Draftsman is stable.** A rigorous suite of tests ensure that Draftsman behaves predictably and correctly (or at least correctly enough), with a code coverage of 100%. Draftsman is verified to work on the latest versions of Python 2 and 3, and is compatible with all versions of Factorio greater than 1.0, with the metrics to prove it.
+* **Draftsman is descriptive.** Draftsman enforces "Factorio-safety" as a core philosophy, which means that if there is any change to a blueprint that would result in an error on import, an exception is thrown. Draftsman also tries to enforce "Factorio-correctness", which means that values that won't break, but are otherwise nonsensical, will raise warnings. Both errors and warnings are verbose, so that any problem with your script can be understood and resolved in seconds.
+* **Draftsman is close-to-source.** Draftsman bases all of its data off of [Wube's `factorio-data` repository](https://github.com/wube/factorio-data), which means that all entities are exactly as you would expect in-game with no inconsistencies. This makes Draftsman up-to-date, makes updating Draftsman for future Factorio versions a breeze, and allows versioning control to monitor changes between Draftsman and Factorio if any future breakage should occur.
+* **Draftsman supports mods.** Draftsman emulates [Factorio's data lifecycle](https://lua-api.factorio.com/latest/Data-Lifecycle.html) directly, meaning that the same loading process that happens when you launch the game is mimicked with a single Draftsman function. In addition to ensuring absolute accuracy to Factorio, this also makes it compatible with custom mod prototypes that can be accessed in script as if they were any other vanilla entity.
+
+Draftsman has custom classes designed around each individual entity and prototype, and is designed to work as seamlessly as possible with blueprint strings and other software. You can import a blueprint string from Factorio and it will be automatically converted into a Draftsman `Blueprint` object with all its functionality, make any change you can think of to it, and then export that `Blueprint` object back into a string to be re-imported into Factorio. Or, you can just make an entirely new `Blueprint` from scratch; Draftsman is designed to work around you, not for you to work around it.
```

Comment:
```suggestion
Draftsman has custom classes designed around each individual entity and prototype, and is designed to work as seamlessly as possible with blueprint strings and other software. You can import a blueprint string from Factorio, and it will be automatically converted into a Draftsman `Blueprint` object with all its functionality, make any change you can think of to it, and then export that `Blueprint` object back into a string to be re-imported into Factorio. Or, you can just make an entirely new `Blueprint` from scratch; Draftsman is designed to work around you, not for you to work around it.
```